src/main/java/ua/yakaboo/pages/HeaderPage.java

```diff
@@ -1,4 +1,126 @@
 package ua.yakaboo.pages;
 
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import static utils.Waits.waitSeconds;
+
 public class HeaderPage {
-}
+    @FindBy(xpath = ".//div[@class='i-logo']//img[contains(@src,'logo_book_cup.png')]")
```

Comment:
Add a line break between class and first web element

Reply:
Added